.gitignore

```diff
@@ -11,4 +11,4 @@ hrabsence/build
 
 # Exclude sass cache from all extension directories
 */.sass-cache/
-
+tags
```

Comment:
tags file are used by the editor to look for definition like "go to declaration"